xapian-bindings/perl/Xapian/DatabaseNotFoundError.pm

```diff
@@ -0,0 +1,11 @@
+package Xapian::DatabaseNotFoundError;
+
+=head1 NAME
+
+Xapian::DatabaseNotFoundError -  DatabaseNotFoundError indicates that a database is not existing.
+
+=head1 DESCRIPTION
+
+
+=cut
+1;
```

Comment:
There's no newline character at the end of this file, as the automated patch checker points out:

https://travis-ci.org/xapian/xapian/jobs/471596630

That's what the two red symbols at the end of the last line in github's diff viewer is trying to tell you.